pkg/cmd/dev/util.go

```diff
@@ -11,68 +11,52 @@
 package main
 
 import (
-	"bufio"
 	"context"
 	"encoding/json"
 	"fmt"
+	"log"
+	"os"
 	"os/exec"
 	"time"
 
-	"github.com/cockroachdb/cockroach/pkg/util/log"
-	"github.com/cockroachdb/redact"
 	"github.com/spf13/cobra"
 )
 
 func mustGetFlagString(cmd *cobra.Command, name string) string {
 	val, err := cmd.Flags().GetString(name)
 	if err != nil {
-		log.Fatalf(context.Background(), "unexpected error: %v", err)
+		log.Fatalf("unexpected error: %v", err)
 	}
 	return val
 }
 
 func mustGetFlagBool(cmd *cobra.Command, name string) bool {
 	val, err := cmd.Flags().GetBool(name)
 	if err != nil {
-		log.Fatalf(context.Background(), "unexpected error: %v", err)
+		log.Fatalf("unexpected error: %v", err)
 	}
 	return val
 }
 
 func mustGetFlagDuration(cmd *cobra.Command, name string) time.Duration {
 	val, err := cmd.Flags().GetDuration(name)
 	if err != nil {
-		log.Fatalf(context.Background(), "unexpected error: %v", err)
+		log.Fatalf("unexpected error: %v", err)
 	}
 	return val
 }
 
 func execute(ctx context.Context, name string, args ...string) error {
 	cmd := exec.CommandContext(ctx, name, args...)
-	log.Infof(ctx, "executing: %s", log.Safe(cmd.String()))
-
-	stdout, err := cmd.StdoutPipe()
-	if err != nil {
-		return err
-	}
-	cmd.Stderr = cmd.Stdout
-
+	log.Printf("executing: %s", cmd.String())
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
 	if err := cmd.Start(); err != nil {
 		return err
 	}
-
-	scanner := bufio.NewScanner(stdout)
-	for scanner.Scan() {
-		line := scanner.Text()
-		log.Infof(ctx, "-- %s\n", redact.Safe(line))
-	}
-	if err := scanner.Err(); err != nil {
-		return err
-	}
 	if err := cmd.Wait(); err != nil {
 		return err
 	}
-
 	return nil
 }
 
```

Comment:
Can't comment at the right spot, but we could standardize around the errors package below, instead of fmt.Errorf:
```diff
-       return "", fmt.Errorf("could not find path to binary %q", target)
+       return "", errors.Newf("could not find path to binary %q", target)
```